src/components/alert/alert.md.scss

```diff
@@ -161,7 +161,7 @@ $alert-md-radio-border-color-on:              $alert-md-button-text-color !defau
 $alert-md-radio-icon-top:                     2px !default;
 
 /// @prop - Left of the icon in the alert radio
-$alert-md-radio-icon-left:                    2px !default;
+$alert-md-radio-icon-start:                    2px !default;
```

Comment:
@AmitMY this variable value is not longer aligned with the rest of scss variables. the same applies many times